Makefile

```diff
@@ -296,6 +302,8 @@ deploy: check-local-context docker-build load-images manifests render-manifests
 	$(KUBECTL) -n kube-system rollout status daemonset kube-multus-ds
 	$(KUBECTL) apply -f $(RENDERED)/whereabouts.yaml
 	$(KUBECTL) -n kube-system rollout status daemonset whereabouts
+	$(KUBECTL) apply -f $(RENDERED)/neonvm-runner-image-loader.yaml
```

Comment:
There also should be similar changes to the `deploy.sh` script we now have. Should it be stored in the `main` branch also?

Can we somehow universalize scripts for local deployments to `k3d`/`kind`, setup in GitHub workflow and deployment to dev/prod?

Reply:
> There also should be similar changes to the `deploy.sh` script we now have

Yup, see neondatabase/cloud#11232.

> Should it be stored in the `main` branch also?

Not yet, I think. Happy to elaborate offline.

> Can we somehow universalize scripts for local deployments to k3d/kind, setup in GitHub workflow and deployment to dev/prod?

We could, yeah. Right now, `deploy.sh` has enough interaction and secondary checks that it'd be annoying for local setup or CI — and we _could_ conditionally deploy without checking, I'm hesitant to make the script more complicated than it needs to be (especially if we're adding a feature that allows bypassing safety checks).